builtin/logical/postgresql/secret_creds.go

```diff
@@ -266,3 +265,16 @@ func (b *backend) secretCredsRevoke(ctx context.Context, req *logical.Request, d
 
 	return resp, nil
 }
+
+// quoteIdentifier quotes an "identifier" (e.g. a table or a column name) to be used as part of an SQL statement.
+//
+// This is a copy of the same function as found in lib/pq (https://github.com/lib/pq/blob/v1.10.4/conn.go#L1640)
```

Comment:
I don't want to be the license party pooper here, but I don't think we can just copy the code without copying the copyright and license.

Or is there another way to accomplish this quoting? It seems a simple enough function.

Reply:
Not at all :) I'll move this out to its own package/file at attach the copyright and license in there.

Reply:
Moved both helpers to the sdk package and attached the copyright and license in [cc7abda](https://github.com/hashicorp/vault/pull/15343/commits/cc7abda403d06f29677e9ad9556c0da070048005).